### ISSUE TITLE
Wrong password bug

### ISSUE THREAD

**Issue by PrzemekSkw** [linuxdeepin/developer-center#3028]
I install Deepin last year on Huawei Matebook 14 and from time to time Deepin forget my password. When I try it few times it shows me info to wait 63 minutes!!!!!
I have to restart to get my password working. It's horrible experience when I have to upgrade something important.
Regards,
Przemek

**Comment by BLumia** [comment]
> forget my password. When I try it few times it shows me info to wait 63 minutes

The retry time will be increased if you retry wrong password for too many times, that's a security thing to prevent more than just brute forcing :/

> It's horrible experience when I have to upgrade something important.

It's your fault for forgetting the password, and it's the login module (PAM)'s responsibility to only allow the person who has the correct password to access the computer.

Learn more about why it works that way and how to configure the related behavior at: https://unix.stackexchange.com/questions/2126/why-is-there-a-big-delay-after-entering-a-wrong-password

**Comment by PrzemekSkw** [comment]
Hi, I don't understand why You close that issue. I don't forget my password. System did. I put wright password. I must restart system to make system accept my password again.
Regards.

**Comment by BLumia** [comment]
> I don't forget my password. System did. I put wright password.

Oh, sorry I might misunderstand what you mean. But I cannot reproduce this issue. Neither this issue under a clean installation nor upgrade from previous version (v20.4). Anyway, I'll reopen this one now.

Since I cannot reproduce this issue, we might need detailed information so we could know what exactly happened. Could you provide the content of `/var/log/auth.log`? Did this happen only once or always happen?

**Comment by tsic404** [comment]
Could you provide `/var/log/auth.log` and `/var/log/daemon.log`?

**Comment by PrzemekSkw** [comment]
Hi, I reinstall system and for now it's OK.
Regards.

**Comment by BLumia** [comment]
Thanks for the update, if you once again got this issue, please open a new issue with the mentioned log files provided so we can look into it, thanks!
